src/ploomber/sources/notebooksource.py

```diff
@@ -606,6 +606,9 @@ def format(self, fmt):
         if extension_changed:
             path = self._path.with_suffix(ext_format)
             Path(self._path).unlink()
+            modified_entry = (Path(entry_point).read_text().replace(
+                f'{self.name}{ext_file}', f'{self.name}{ext_format}'))
```

Comment:
in the rare event that `{self.name}{ext_file}` does not appear in the pipeline.yaml content, I think we should show a warning message as well (click.secho). this might happen if the user has an `env.yaml` file

```yaml
path: script.py
```

and in the pipeline.yaml, it has the `{{script}}` placeholder.

Reply:
to test that the warning message appears you can use capsys https://docs.pytest.org/en/6.2.x/capture.html

and pass the `--entry-point test_pkg.entry.plain_function` argument